Earthfile

```diff
@@ -20,11 +20,26 @@ proto:
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf mod update
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf build
   RUN --mount=type=cache,target=$BUF_CACHE_DIR buf generate
+  
+  RUN sed -i'.bu' '/client.UploadTranslationFile/i \
+  \\tfile, _, err := req.FormFile("file")\n\
+	\t\tif err != nil {\n\
+		\t\t\treturn nil, metadata, status.Errorf(codes.InvalidArgument, "%v", err)\n\
+	\t\t}\n\
+	\n\
+	\tdata, err := io.ReadAll(file)\n\
+	\t\tif err != nil {\n\
+		\t\t\treturn nil, metadata, status.Errorf(codes.InvalidArgument, "%v", err)\n\
+	\t\t}\n\
+	\n\
+	\tprotoReq.Data = data\n' gen/proto/go/translate/v1/translate.pb.gw.go
+
+  RUN rm gen/proto/go/translate/v1/translate.pb.gw.go.bu
```

Comment:
That is the best solution I could think of. So bassicaly we search and add code, in generated code by grpc-gateway.  That saves us writing custom handlers, and curl is executed using form data e.g.:
```curl
curl --location --request PUT 'http://localhost:8080/v1/files/' \
--form 'file=@"/Users/vladislavs/Projects/translate/example.translate.json"'
```

Reply:
We need a comment to describe why you do it.

Reply:
Basically, I have identified 4 options:

1. Develop a middleware that reads the file from the form-data and embeds it in the request body (not feasible due to various limitations, primarily related to the grpc-gateway implementation).
2. Divide the upload request into two endpoints, one public and one internal. The idea is that the client sends a request to the external endpoint with the file in the form-data, then that endpoint reads the file and makes a request to the internal endpoint. (This would require custom handlers and could cause issues with traces in the future, and would result in a total of three calls instead of two).
3. Similar to option 2, but with only one endpoint for the REST API. This would read the file from the form-data and then send the request in gRPC (which would require developing a custom handler and a client for gRPC for just one use case, which seemed overly cumbersome).
4. Modifying the grpc-gateway REST API implementation by injecting code that reads explicitly from the form-data and embeds it in protoReq.Data. (This is the best option as it does not require custom handlers, however, it could be potentially unstable if the grpc-gateway changes its generator in the future)."